### DISCUSSION TITLE
Add possibility to reload config

### DISCUSSION THREAD

**Discussion by ysechynskyy** [wg-easy/wg-easy#374] [Ideas]
I use vpn in docker compose on 2 ec2 instances mounted between by efs. VPN available Thought AWS NETWORK LOAD BALANCER. If i add user on master node , slave will be synced only after restart the docker compose.(Or i can run inside docker wg syncconf wg0 <(wg-quick strip wg0) command, so it allows to connect the new client but ui will still show old user list)

Can you add something like reload config and UI or something?

**Comment by pheiduck** [comment]
@ysechynskyy Feel free to create a PR which implment this feature.
